io/pointer/pointer.go

```diff
@@ -36,6 +36,10 @@ type Event struct {
 	Position f32.Point
 	// Scroll is the scroll amount, if any.
 	Scroll f32.Point
+	// Scale is the rate of a pinch gesture
+	Magnification float32
```

Comment:
"Rate" doesn't sound right. Better to say

// Magnification is the change of scale from a pinch gesture.